src/accordian/Accordion.tsx

```diff
@@ -55,29 +54,30 @@ export class Accordion extends React.Component<AccordionProps, AccordionState> {
         }
     }
 
-    handleButtonClick = (index: any, accordionMultiPanel: any) => {
-        let items = this.state.panelItems;
-        const {prevItemIndex} = this.state;
-        if (prevItemIndex != -1 && prevItemIndex === index) {
-            items[index].isOpen = false;
-            this.setState({prevItemIndex: -1});
-            items[prevItemIndex].content = this.getItemContent(prevItemIndex, items[prevItemIndex].title, true);
-            this.setState({panelItems: items});
-        } else if (items[index].isOpen === true) {
-            items[index].isOpen = false;
-            items[index].content = this.getItemContent(index, items[index].title, true);
-            this.setState({panelItems: items});
-        } else {
-            if (!accordionMultiPanel && prevItemIndex != -1) {
-                items[prevItemIndex].isOpen = false;
-                items[prevItemIndex].content = this.getItemContent(prevItemIndex, items[prevItemIndex].title, true);
+    handleButtonClick = (clickedItemIndex: number, accordionMultiPanel: boolean | undefined) => {
```

Comment:
make `accordionMultiPanel` as optional parameter or assigned it for false if user has not provided any value